#This is a message to store magnetometer values on x, y, and z axes


float64 magX
float64 magY
float64 magZ
